client/me/account/main.jsx

```diff
@@ -955,30 +955,32 @@ class Account extends Component {
 
 						{ this.props.canDisplayCommunityTranslator && this.communityTranslator() }
 
-						<FormFieldset
-							className="account__settings-admin-home"
-							style={
-								config.isEnabled( 'layout/site-level-user-profile' ) ? { marginBottom: 0 } : {}
-							}
-						>
+						<FormFieldset className="account__settings-admin-home">
 							<FormLabel id="account__default_landing_page">
 								{ translate( 'Admin home' ) }
 							</FormLabel>
 							<ToggleSitesAsLandingPage />
 						</FormFieldset>
 
 						{ config.isEnabled( 'me/account/color-scheme-picker' ) &&
-							! config.isEnabled( 'layout/site-level-user-profile' ) &&
 							supportsCssCustomProperties() && (
 								<FormFieldset>
 									<FormLabel id="account__color_scheme" htmlFor="color_scheme">
 										{ translate( 'Dashboard color scheme' ) }
 									</FormLabel>
-									<ColorSchemePicker
-										disabled={ this.getDisabledState( INTERFACE_FORM_NAME ) }
-										defaultSelection="classic-dark"
-										onSelection={ this.updateColorScheme }
-									/>
+									{ config.isEnabled( 'layout/site-level-user-profile' ) ? (
+										<FormSettingExplanation>
+											{ translate(
+												'You can now set the color scheme on your individual site by visiting Users → Profile from your site dashboard.'
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)
Maybe we can show the "site selector modal" when clicking "visiting Users → Profile"?

Reply:
I was thinking about that as well. But, I decided not to do it because I thought this is just a temporary notice. The whole section should be removed after some time.